packages/remix/src/index.server.ts

```diff
@@ -133,9 +167,14 @@ export function init(options: RemixOptions): void {
     return;
   }
 
-  instrumentServer();
+  options.defaultIntegrations = [
+    ...getRemixDefaultIntegrations(options as NodeOptions),
```

Comment:
m: Could we just move this into `getRemoxDefaultIntegrations`? That already receives the options as arguments, so we can conditionally add `remixINtegration` in there and not do anything here?